src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -113,6 +116,9 @@ public static class DnnCatalog
             int epoch = 100,
             int batchSize = 10,
             float learningRate = 0.01f,
+            bool enableEarlyStopping = true,
+            float earlyStoppingminDelta = 0.01f,
```

Comment:
`earlyStoppingminDelta` - I assume `Min` is a new word and should be capitalized.